toolkit/imageconfigs/osguard-arm64.yaml

```diff
@@ -0,0 +1,247 @@
+storage:
+  bootType: efi
+
+  disks:
+    - partitionTableType: gpt
+      partitions:
+        - id: esp
+          type: esp
+          label: esp
+          size: 512M
+
+        - id: boot-a
+          type: linux-generic
+          label: boot-a
+          size: 128M
+
+        - id: usr-a
+          type: linux-generic
+          size: 1G
+
+        - id: usr-hash-a
+          type: usr-verity
+          size: 128M
+
+        - id: root-a
+          type: root
+          label: root-a
+          size: 4G
+
+  verity:
+    - id: usrverity
+      name: usr
+      dataDeviceId: usr-a
+      hashDeviceId: usr-hash-a
+      dataDeviceMountIdType: uuid
+      hashDeviceMountIdType: uuid
+
+  filesystems:
+    - deviceId: esp
+      type: fat32
+      mountPoint:
+        idType: part-label
+        path: /boot/efi
+        options: umask=0077,noexec,nodev,nosuid
+
+    - deviceId: boot-a
+      type: ext4
+      mountPoint:
+        idType: uuid
+        path: /boot
+        options: noexec,nodev,nosuid
+
+    - deviceId: usrverity
+      type: ext4
+      mountPoint:
+        path: /usr
+        options: ro,nodev
+
+    - deviceId: root-a
+      type: ext4
+      mountPoint:
+        path: /
+        options: nodev,nosuid,x-initrd.mount,x-systemd.growfs
+
+os:
+  bootloader:
+    resetType: hard-reset
+  hostname: azure-linux-os-guard
+
+  selinux:
+    mode: permissive
+
+  uki:
+    kernels: auto
+  
+  kernelCommandLine:
+    extraCommandLine:
+      - console=tty0
+      - console=tty1
+      - console=ttyS0
+      - rd.luks=0
+      - rd.hostonly=0
+      - ipe.enforce=0
+      - fips=1
+      - console=ttyAMA0
+      - earlycon=pl011,0xeffec000
+      - initcall_blacklist=arm_pmu_acpi_init
```

Comment:
Taken from Marketplace-gen2 config.

_Copilot explanation:_
This prevents the kernel from calling the arm_pmu_acpi_init initialization function during boot. This is to disable a specific ARM performance monitoring unit initialization routine, likely to avoid hardware conflicts or boot issues.